unpackaged/pre/opportunity_record_types/package.xml

```diff
@@ -4,5 +4,5 @@
     <members>*</members>
     <name>CustomObject</name>
   </types>
-  <version>29.0</version>
+  <version>42.0</version>
```

Comment:
would you revert these back to 29, since we aren't changing them?

Reply:
Will do